features/bootstrap/FeatureContext.php

```diff
@@ -34,6 +34,21 @@ public function prepareProcess()
         $this->process = new Process(null);
     }
 
+
+    /**
+     * @Given I have not configured behat to use shared kernel
+     */
+    public function iHaveNotConfiguredBehatToUseSharedKernel()
+    {
```

Comment:
Can you simply assert that the `symfony2_extension.kernel.shared` parameter has the correct value?

Reply:
yes, good point.